Add DVC for VS Code Extension sorting demo

julieg18 opened the pull request:
* As discussed in planning, adds a mention of sorting in the vscode extension to the docs

## Demo

In `/doc/command-reference/exp/show`

~https://github.com/iterative/dvc.org/assets/43496356/94663389-820d-485e-9982-c07ed3c6f7f8~

<img width="1265" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/c753979a-2c52-433f-a94d-b13ed5b98afb">

In `/doc/user-guide/experiment-management`:

<img width="1312" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/03b2a13c-5193-461c-b2b7-68a5d9f5fee8">

julieg18 commented on content/docs/command-reference/exp/show.md:
````diff
@@ -281,6 +281,18 @@ $ dvc exp show --only-changed --sort-by=roc_auc --sort-order desc
  ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 ```
 
+<details>
````
Another alternative place I considered was somewhere in https://dvc.org/doc/user-guide/experiment-management/comparing-experiments.

dberenbaum replied:
Why not both? 😄 It's a very small addition, so I don't see much harm including it there also.

julieg18 replied:
Updated `comparing-experiments.md` as well! 

<img width="1312" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/f41a00b4-4fc9-4fbb-8b14-d0592057803e">

julieg18 commented:
@dberenbaum, is this ready to merge or are there more changes needed?

dberenbaum reviewed (changes requested):
Thanks for the ping @julieg18!

In the exp show page, can we move it above `Paginating the output` near the top right after the text below?

> Experiments in the table are first grouped (by parent commit). They are then sorted inside each group, chronologically by default. The --sort-by and --sort-order options can change this ordering, based on any single, visible metric or param.

That will make it more visible.

julieg18 commented:
> In the exp show page, can we move it above Paginating the output near the top right after the text below?

Moved! Did you want to keep things wrapped in a details block? I removed it for now: 

<img width="1265" alt="image" src="https://github.com/iterative/dvc.org/assets/43496356/d3fef710-470a-49f4-970d-482ba9bb28fa">

dberenbaum reviewed (approved):
Looks good @julieg18 !
